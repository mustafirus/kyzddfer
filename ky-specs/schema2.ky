rack ver(1.0)
  tables
    validation_test
      # Перевірка "сирого" рядка для регулярного виразу
      email varchar(255) !required validate_re([[\w+@\w+\.\w+]])
  apps
