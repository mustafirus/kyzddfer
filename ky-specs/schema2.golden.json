{
  "rack": {
    "tables": [
      {
        "name": "validation_test",
        "fields": [
          {
            "name": "email",
            "type": "varchar(255)",
            "attrs": {
              "validate_re": "\\w+@\\w+\\.\\w+"
            },
            "flags": [
              "required"
            ]
          }
        ]
      }
    ]
  }
}